### ISSUE TITLE
IDE: Sort editor scripts commands alphabetically

### ISSUE THREAD

**Issue by aglitchman** [defold/defold#8902]
**Is your feature request related to a problem? Please describe (REQUIRED):**
I actively use Editor Scripts to create my own commands and the IDE often displays them out of order.

**Describe the solution you'd like (REQUIRED):**
 It would be great if it sorted them alphabetically by name.

**Describe alternatives you've considered (REQUIRED):**
N/A

**Additional context (OPTIONAL):**
In this screenshot the commands are in random order, but in the editor script they come one after the other, alphabetically.
![image](https://github.com/defold/defold/assets/193258/f27e169f-f88b-45a7-80ea-0543ae46c7ea)

I couldn't reproduce this on an isolated example, but I still attached for ease of development for whoever implements this in the future 🙌

[issue_editor_scripts_random.zip](https://github.com/defold/defold/files/15266167/issue_editor_scripts_random.zip)

**Comment by vlaaad** [comment]
Good idea! Also, I just thought it would be nice if we also supported nested menus in extensions, so you could have a single "Convert to Ogg Vorbis" menu item with a drop down that suggests different conversion options. Do you think that would be useful for you?

**Comment by aglitchman** [comment]
> I just thought it would be nice if we also supported nested menus in extensions, so you could have a single "Convert to Ogg Vorbis" menu item with a drop down that suggests different conversion options. Do you think that would be useful for you?

Yes, that would be nice! I have a couple of commands that I have “grouped” for myself using a prefix: “Extension Name: Do That” 

P.S. I was thinking that commands don't necessarily need to be sorted alphabetically. My request is more about the fact that they are sometimes randomized currently (so, is it a bug?).
